system/Validation/Validation.php

```diff
@@ -812,24 +831,21 @@ protected function splitRules(string $rules): array
 	{
 		$nonEscapeBracket = '((?<!\\\\)(?:\\\\\\\\)*[\[\]])';
 		$pipeNotInBracket = sprintf(
-				'/\|(?=(?:[^\[\]]*%s[^\[\]]*%s)*(?![^\[\]]*%s))/',
-				$nonEscapeBracket,
-				$nonEscapeBracket,
-				$nonEscapeBracket
+			'/\|(?=(?:[^\[\]]*%s[^\[\]]*%s)*(?![^\[\]]*%s))/',
+			$nonEscapeBracket,
+			$nonEscapeBracket,
+			$nonEscapeBracket
 		);
 
-		$_rules = preg_split(
-				$pipeNotInBracket,
-				$rules
-		);
+		$rules = preg_split($pipeNotInBracket, $rules);
 
-		return array_unique($_rules);
+		return array_unique($rules);
```

Comment:
iirc, it was @michalsn 's request to keep `$_` naming, see https://github.com/codeigniter4/CodeIgniter4/blob/63d0b27420e7506f3ac84147a3d7bc6c0adc6e39/utils/Rector/UnderscoreToCamelCaseVariableNameRector.php#L107